src/mount-support.c

```diff
@@ -392,3 +392,23 @@ static char * __attribute__ ((used))
 	*offsetp = offset;
 	return (offset < fulllen) ? &path[offset] : NULL;
 }
+
+/*
+ * Check that @subdir is a subdir of @dir.
+ * @len is the length of @dir (to avoid having to recalculate it).
+**/
+static bool __attribute__ ((used))
+    is_subdir(const char *subdir, const char *dir, size_t len)
```

Comment:
How to use this function is not obvious from its signature. You documented this (thank you), but perhaps it would be more clear to accept a length for both of these?